skimage/transform/_geometric.py

```diff
@@ -816,13 +816,21 @@ class AffineTransform(ProjectiveTransform):
     Has the following form::
 
         X = a0*x + a1*y + a2 =
-          = sx*x*cos(rotation) - sy*y*sin(rotation + shear) + a2
+          = sx*x*cos(rotation) - sy*y*(tan(shear) * sin(rotation) + sin(rotation)) + a2
 
         Y = b0*x + b1*y + b2 =
-          = sx*x*sin(rotation) + sy*y*cos(rotation + shear) + b2
+          = sx*x*sin(rotation) - sy*y*(tan(shear) * sin(rotation) - cos(rotation)) + b2
```

Comment:
Modifying @stefanv's snippet according to https://github.com/scikit-image/scikit-image/pull/6717#issuecomment-1496877600, I get the following:

```python
import sympy as sp

s_x, s_y, theta, phi_x, h_x, h_y, x, y = sp.var(['S_x', 'S_y', 'theta', 'phi_x', 'h_x', 'h_y', 'x', 'y'])

scale = sp.Matrix([
    [S_x, 0, 0],
    [0, S_y, 0],
    [0, 0, 1],
])
shear = sp.Matrix([
    [1, -sp.tan(phi_x), 0],
    [0, 1, 0],
    [0, 0, 1],
])
rotate = sp.Matrix([
    [sp.cos(theta), -sp.sin(theta), 0],
    [sp.sin(theta), sp.cos(theta), 0],
    [0, 0, 1],
])

p = sp.Matrix([[x], [y], [1]])

rotate * shear * scale * p  # gives
# [S_x*x*cos(theta) + S_y*y*(-sin(theta) - cos(theta)*tan(phi_x))]
# [S_x*x*sin(theta) + S_y*y*(-sin(theta)*tan(phi_x) + cos(theta))]
```

It seems to me that the only difference is the `cos(theta)*tan(phi_x)` / `tan(shear) * sin(rotation)` part in the first equation. You actually got this result in your original description https://github.com/scikit-image/scikit-image/pull/6717#issue-1564229575: the last part of the rendered equation seems correct while you than don't carry over the cos... :thinking: